Client/Ecosia/UI/NTP/Cells/NTPBookmarkNudgeCell.swift

```diff
@@ -0,0 +1,182 @@
+
+            badge.heightAnchor.constraint(equalToConstant: UX.badgeHeight).priority(.required),
+            badge.topAnchor.constraint(equalTo: backgroundCard.topAnchor, constant: UX.insetMargin),
+            badge.leadingAnchor.constraint(equalTo: backgroundCard.leadingAnchor, constant: UX.insetMargin),
+            
+            closeButton.widthAnchor.constraint(equalToConstant: 44),
+            closeButton.heightAnchor.constraint(equalToConstant: 44),
+            closeButton.trailingAnchor.constraint(equalTo: backgroundCard.trailingAnchor),
+            closeButton.topAnchor.constraint(equalTo: backgroundCard.topAnchor),
+            
+            descriptionLabel.topAnchor.constraint(equalTo: badge.bottomAnchor, constant: UX.insetMargin / 2),
+            descriptionLabel.leadingAnchor.constraint(equalTo: backgroundCard.leadingAnchor, constant: UX.insetMargin),
+            descriptionLabel.trailingAnchor.constraint(equalTo: icon.leadingAnchor, constant: -UX.insetMargin),
+            
+            openBookmarksButton.heightAnchor.constraint(equalToConstant: 32).priority(.required),
+            openBookmarksButton.topAnchor.constraint(equalTo: descriptionLabel.bottomAnchor, constant: UX.insetMargin),
+            openBookmarksButton.leadingAnchor.constraint(equalTo: backgroundCard.leadingAnchor, constant: UX.insetMargin),
+            openBookmarksButton.bottomAnchor.constraint(equalTo: backgroundCard.bottomAnchor, constant: -UX.insetMargin),
+            
+            icon.trailingAnchor.constraint(equalTo: backgroundCard.trailingAnchor, constant: -UX.insetMargin),
+            icon.bottomAnchor.constraint(equalTo: openBookmarksButton.bottomAnchor, constant: 0),
+            icon.widthAnchor.constraint(equalToConstant: 64).priority(.required),
+            icon.heightAnchor.constraint(equalToConstant: 64).priority(.required),
+            
+            backgroundCard.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -UX.insetMargin)
+        ])
+        
+        closeButton.addTarget(self, action: #selector(handleClose), for: .touchUpInside)
+        openBookmarksButton.addTarget(self, action: #selector(handleOpenBookmarks), for: .touchUpInside)
+    }
+    
+    func applyTheme() {}
+    
+    override func preferredLayoutAttributesFitting(_ layoutAttributes: UICollectionViewLayoutAttributes) -> UICollectionViewLayoutAttributes {
+            let targetSize = CGSize(width: layoutAttributes.frame.width, height: 0)
+            layoutAttributes.frame.size = contentView.systemLayoutSizeFitting(targetSize, withHorizontalFittingPriority: .required, verticalFittingPriority: .fittingSizeLevel)
+            return layoutAttributes
+    }
+    
+    @objc private func handleOpenBookmarks() {
+        openBookmarksHandler?()
+    }
+    
+    @objc private func handleClose() {
+        closeHandler?()
+    }
+}
+
+private class NTPBookmarkNudgeCellBadge: UIView {
```

Comment:
Apologize in advance if you were planning to do this anyway.
Leaving it as "reminder". What do you think about moving this class elsewhere?

```suggestion
private final class NTPBookmarkNudgeCellBadge: UIView {
```

Reply:
Yep, totally makes sense to me! Updating.